### ISSUE TITLE
FILE NOT FOUNDERROR

### ISSUE THREAD

**Issue by lszcaption** [ruotianluo/self-critical.pytorch#233]
hi，when I run : python tools/evals.py --model model.pth --infos_path infos.pkl --image_folder blah --num_images 10.it tell me no such file or directory:"infos.pkl".so I find csdn tell me that 如果文件夹路径不存在，则创建文件夹路径.
def __init__(self,path='./'):
       #save the path
       self.path=path
       if not os.path.exists(path):
              os.mkdir(path)
and then I do the same,but still have this error. would you tell me how to solve it?

**Comment by ruotianluo** [comment]
这个名字要换的，换成你训练出来的模型的model.pth 和infos.pkl，或者你下载model zoo里面的

**Comment by lszcaption** [comment]
yes,my file name is infos_fc.pkl.and I do the same as u say,but,still the error???
